vertx-web-client/src/main/java/io/vertx/ext/web/client/HttpResponse.java

```diff
@@ -119,20 +120,29 @@
    */
   @CacheReturn
   @Nullable
-  String bodyAsString();
+  default String bodyAsString() {
```

Comment:
I have just rebased it. This was part of the original branch https://github.com/vert-x3/vertx-web/commit/8811f365844e1ba4ae18ea92b479f44d00c947bc#diff-4a9e17d3df50140ed8dc9bb8ab1411a6R132